docs/presentation/README.md

```diff
@@ -1,97 +1,101 @@
 
-## Popularité
-Vue.js a battu des records en termes de hausse de popularité depuis 2015. Il s'agit aujourd'hui du 2ème projet en nombre de stars sur Github. Malgré l'absence de financement, de nombreux meetups et conférences se sont auto-organisés dans différents pays. Il s'agit de l'un des succès les plus retentissants dans le monde open-source depuis ces cinq dernières années.
+![Github stars development between React, Vue and Angular](../assets/popularite.jpg)
 
-Le secret de son succès ? Rien de révolutionnaire : un focus sur la simplicité, une courbe d'apprentissage très faible, une excellente documentation et une communauté à l'écoute.
+## Comparison between the frameworks
 
-![Evolution des stars Github entre React, Vue et Angular](../assets/popularite.jpg)
+Let's try to compare Vue with the other two most popular JS frameworks, React and Angular.
 
-## Comparaison avec les autres frameworks
-Tâchons de comparer Vue avec les deux autres frameworks JS les plus populaires, React et Angular.
+![React, Vue and Angular](../assets/react-vue-angular.jpg)
 
-![React, Vue et Angular](../assets/react-vue-angular.jpg)
+### Similarity between these 3 frameworks:
 
-### Points communs entre ces 3 frameworks:
-- Très populaires, utilisé par de grosses entreprises
-- Matures, stables, support à long terme financé
-- Codebase orientée composants
-- Adapté aux stacks modernes (ES6+ / TypeScript)
-- Large écosystème de composants et d’outillage
+- Very popular, used by top companies
+- Mature, stable, funded long-term support
+- Component oriented codebase
+- Suitable for modern technological stacks (ES6+ / TypeScript)
+- Large ecosystem of components and tools
 
-### Positionnement / autodescription
+### Product positioning / self-description
 
-![Positionnement de React, Vue et Angular](../assets/autodescription.jpg)
+![Product positioning of React, Vue and Angular](../assets/autodescription.jpg)
 
-React, Vue et Angular se positionnent dans des catégories différentes et cela s'observe dès les premières lignes d'introduction. React se définit comme une **bibliothèque** indépendante du reste de la stack technique. Angular se définit comme **le framework** unique répondant à tous les besoins, desktop comme mobile. Tandis que Vue se positionne entre deux comme un **framework progressif**, polyvalent et que l'on peut adopter par étapes. Un juste milieu entre une bibliothèque et un framework tout équipé.
+
+React, Vue and Angular position themselves in different categories, this can be observed from the first lines of their introduction. React is defined as a **library**, independent of the technical stack. Angular is defined as the one **framework** for all needs, desktop and mobile. While Vue is caught in the middle, defined as a **progressive framework**, versatile which can be adopted step-by-step. A perfect middle ground between a library and a fully equipped framework.
```

Comment:
```suggestion
React, Vue and Angular position themselves in different categories, this can be observed from the first lines of their introduction. React is defined as a **library**, independent of the technical stack. Angular is defined as the one **framework** for all needs, desktop and mobile. While Vue is caught in the middle, defined as a **progressive framework**, versatile and **incrementally adoptable**. A perfect middle ground between a library and a fully equipped framework.
```